heka/map.jinja

```diff
@@ -14,6 +14,8 @@ Debian:
   splitter: {}
   encoder: {}
   decoder: {}
+  extra_fields:
+    environment_label: {{ grains.domain }}
```

Comment:
I think we should modify the README to show how to add extra fields, but this can be done with a separate formula.